### PR TITLE
Fix ``get_object`` call which was missing request

### PR DESCRIPTION
**Pull request by marksweb**
## Description

In `get_content_obj` the first call is to `get_object` is missing an argument.

```
    def edit_title_fields(self, request, page_id, language):
        page = self.get_object(page_id)
```

But `get_object` takes the request as the first argument and then the object ID as the second.

## Related resources

* Django source; https://github.com/django/django/blame/main/django/contrib/admin/options.py#L851

## Checklist

* [x] I have opened this pull request against ``develop-4``
* [x] I have added or modified the tests when changing logic
* [x] I have followed [the conventional commits guidelines](https://www.conventionalcommits.org/) to add meaningful information into the changelog
* [x] I have read the [contribution guidelines ](https://github.com/django-cms/django-cms/blob/develop/CONTRIBUTING.rst) and I have joined #workgroup-pr-review on [Slack](https://www.django-cms.org/slack) to find a “pr review buddy” who is going to review my pull request.

### CODE REVIEW

File: cms/admin/pageadmin.py [hunk 1]
```diff
@@ -705,16 +705,16 @@ def copy_page(self, request, page_id):
...
-                                   "translated in any of the languages configured by the target site."))
+                                  "translated in any of the languages configured by the target site."))
             return jsonify_request(HttpResponseBadRequest(message))
 
         new_page = form.copy_page()
         return HttpResponse(json.dumps({"id": new_page.pk}), content_type='application/json')
 
     def edit_title_fields(self, request, page_id, language):
-        page = self.get_object(page_id)
+        page = self.get_object(request, page_id)
```
Comment (Aiky30) [inline cms/admin/pageadmin.py:717]:
```suggestion
        page = self.get_object(request, object_id=page_id)
```

The above suggestion will keep the fix consistent with all other use cases of get_object on this page.

**Review by Aiky30 — CHANGES_REQUESTED** [review]
Minor amendment required and requires changelog entry.

File: cms/admin/pageadmin.py (hunk 1 above)
Comment (marksweb) [reply cms/admin/pageadmin.py:717]: @Aiky30 Well spotted!

**Comment by marksweb** [comment]
@Aiky30 Changelog updates are something I really want to get automated. The way github releases generate them is really good.

**Comment by Aiky30** [comment]
> @Aiky30 Changelog updates are something I really want to get automated. The way github releases generate them is really good.

Completely agree, I think this was already implemented in cms 3 develop, need to put in the time to pull it out and into 4.

**Comment by marksweb** [comment]
@Aiky30 Yeah v3 generates the changelog using this script executed from the build script;
https://github.com/django-cms/django-cms/blob/develop/scripts/make-changelog

I really want to break up the build script into the necessary parts to try to build releases via GH actions. As I mentioned last Friday, we've got releases ready to go but we can't run the script.

**Pull request merged**